packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html

```diff
@@ -9,71 +9,79 @@
         </button>
     </div>
     <!-- subridg -->
-    <li
-        class="vrack__assigned-ipv6__subrange"
-        data-ng-repeat="bridged in $ctrl.bridgedSubranges track by $index"
-    >
-        <span
-            class="vrack__assigned-ipv6__subrange__title"
-            data-ng-bind="::bridged.bridgedSubrange"
-        ></span>
-        <span>
-            {{:: $ctrl.label.SLAAC }}
-            <oui-switch
-                data-ng-hide="bridged.loading"
-                model="bridged.model"
-                on-change="$ctrl.toggleSubrange(bridged)"
-            >
-            </oui-switch>
-            <oui-spinner data-size="s" data-ng-show="bridged.loading">
-            </oui-spinner>
-        </span>
-        <button
-            type="button"
-            class="oui-popover-button"
-            data-oui-popover="{{:: 'vrack__assigned-ipv6__subrange_slaac_help' | translate }}"
-            data-oui-popover-placement="right"
-            data-oui-popover-scope="$ctrl"
-            data-oui-popover-template="vrack__assigned-ipv6__subrange_slaac_help.html"
-        ></button>
-        <script
-            type="text/ng-template"
-            id="vrack__assigned-ipv6__subrange_slaac_help.html"
+    <ul class="vrack__assigned-ipv6__sub-item">
+        <li
+            class="vrack__assigned-ipv6__subrange"
+            data-ng-repeat="bridged in $ctrl.bridgedSubranges track by $index"
```

Comment:
I thought there was only 1 bridged subrange ?

Reply:
based on Api we can have many subrange just when we add ipv6 to vrack first one created by default

Reply:
Ok weird but if it is possible from the API let's keep  it like this :)